spreadsheets/collision_rank.yaml

```diff
@@ -0,0 +1,943 @@
+# reserved for UX/UI
+- _reserved: {from: 1, to: 100}
+# reserved for house theme overlays
+- _reserved: {from: 101, to: 201}
+# reserved for data overlays
+- _reserved: {from: 202, to: 302}
+- {layer: 'earth', kind: 'continent'}
+- {layer: 'water', kind: 'ocean'}
+- {layer: 'places', kind: 'country'}
+# reserved for population_rank
+- _reserved: {from: 306, to: 325}
```

Comment:
I think all the reserved lines from here down should be reserved counts instead of numerical ranges. Here this would be `reserved: { count: 10 }`

Reply:
Support for reserved counts added in 82d74dc.